plugin.xml

```diff
@@ -76,11 +76,31 @@
         <config-file target="config.xml" parent="/*">
             <feature name="InAppBrowser">
                 <param name="ios-package" value="CDVInAppBrowser" />
+                <param name="onload" value="true" />
```

Comment:
What does this `onload` param do?

Reply:
`onload` specifies whether to initialize the plugin when the app controller initializes. There's some [docs about it here](https://cordova.apache.org/docs/en/latest/config_ref/index.html#feature).
It ensures that [`pluginInitialize()`](https://github.com/dpa99c/cordova-plugin-inappbrowser/blob/CB-7179/src/ios/CDVInAppBrowser.m#L31) is called on app startup, rather than the first plugin invocation.

Reply:
I learned something, thanks ;)

Why this change?

Reply:
It just ensures that the default values defined by [`pluginInitialize()`](https://github.com/dpa99c/cordova-plugin-inappbrowser/blob/CB-7179/src/ios/CDVInAppBrowser.m#L31) are set right away at app startup. 
I don't remember the exact reason it was needed (it's been a few months since I created this PR) but I'm pretty sure it was necessary.
The change is minimal and will not cause side effects.